lib/util.js

```diff
@@ -196,3 +197,27 @@ module.exports.processDate = function processDate(inputDate) {
     return date;
   }
 };
+
+module.exports.handleInput = function (input, cwd) {
```

Comment:
Can you add a comment for what this does? since handleInput is vague. looks like it takes the CLI input and resolves a set of project paths to be operated against.

Reply:
Took a stab, let me know!